client/controllers/downloadCSVModal.coffee

```diff
@@ -1,10 +1,29 @@
+Template.downloadCSVModal.onCreated ->
+  @prepared = new ReactiveVar(null)
+  @preparing = new ReactiveVar(false)
```

Comment:
The it looks like the preparing variable isn't necessary since it could be derived from the prepared variable.